crowbar_framework/config/routes.rb

```diff
@@ -17,10 +17,15 @@
 
 Rails.application.routes.draw do
   apipie
-  # Install route from each barclamp
-  Rails.root.join("config", "routes.d").children.each do |routes|
-    eval(routes.read, binding) if routes.extname == ".routes"
-  end if Rails.root.join("config", "routes.d").directory?
+
+  route_dirs = [Pathname.new("/etc/crowbar/routes.d/"), Rails.root.join("config", "routes.d")]
```

Comment:
not sure if the routes would end up in the backup. I guess that would be wrong then